cmd/antrea-controller/controller.go

```diff
@@ -89,7 +95,11 @@ func run(o *Options) error {
 
 	go networkPolicyController.Run(stopCh)
 
-	go apiServer.GenericAPIServer.PrepareRun().Run(stopCh)
+	preparedAPIServer := apiServer.GenericAPIServer.PrepareRun()
+	antctl.CommandList.ApplyToMux(preparedAPIServer.GenericAPIServer.Handler.NonGoRestfulMux, nil, controllerMonitor)
```

Comment:
Maybe add a comment here to show that it is for remote mode? Also, you can point out that CLI supports remote mode for controller here.

Reply:
Added.